src/Umbraco.Web.UI.Client/src/common/resources/redirecturls.resource.js

```diff
@@ -0,0 +1,127 @@
+/**
+ * @ngdoc service
+ * @name umbraco.resources.redirectUrlResource
+ * @function
+ *
+ * @description
+ * Used by the redirect url dashboard to get urls and send requests to remove redirects.
+ */
+(function() {
+    'use strict';
+
+    function redirectUrlsResource($http, umbRequestHelper) {
+
+        var redirectBaseUrl = "backoffice/api/RedirectUrlManagement/";
```

Comment:
This should not be coded like this, the URL needs to come from the server like we do with all other resources. All requests need to be done consistently like all of our other resources.